emcache/base.py

```diff
@@ -311,6 +311,22 @@ async def gats_many(self, exptime: int, keys: Sequence[bytes], return_flags=Fals
         gats <exptime> <key>*\r\n
         """
 
+    @abstractmethod
```

Comment:
Same here, if you don't mind lets focus on making one change at a time. Lets focus on the `stats` command

Reply:
Ok